packages/orchestration/src/orchestration-types.ts

```diff
@@ -56,3 +59,39 @@ export interface OrchestrationModuleConfig {
    */
   grounding?: GroundingModuleConfig;
 }
+
+/**
+ * Request options for orchestration.
+ */
+export interface RequestOptions {
+  /**
+   * Prompt configuration.
+   */
+  prompt?: Prompt;
+  /**
+   * Custom request configuration.
+   */
+  requestConfig?: CustomRequestConfig;
+  /**
+   * Whether to stream the response.
+   */
+  stream?: boolean;
+  /**
+   * Options for the stream.
+   */
+  streamOptions?: StreamOptions;
+}
+
+/**
+ * Options for the stream.
+ */
+export interface StreamOptions extends GlobalStreamOptions {
```

Comment:
[pp] Or what about having an additional attribute called `global` to store chunk_size etc. Just in case in the future we have more and more, and we need to manually extract the global options one by one.

Reply:
Yes!!